app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
There is no validation for the command

Command name should be exactly "cp" and command should consist of 3 parts